docs/reference/docs/reindex.asciidoc

```diff
@@ -136,7 +136,7 @@ POST _reindex
 // TEST[setup:twitter]
 
 You can limit the documents by adding a type to the `source` or by adding a
-query. This will only copy ++tweet++&apos;s made by `kimchy` into `new_twitter`:
+query. This will only copy ++tweet++s made by `kimchy` into `new_twitter`:
```

Comment:
I think we can even remove the special formatting for "tweet". I think it was useful when this was still refering to the "tweet" type but since the example nor uses "_doc" it looks better in the result to simply write "tweets", or what do you think?